conda_smithy/linter/utils.py

```diff
@@ -46,6 +54,18 @@
 CONDA_BUILD_TOOL = "conda-build"
 RATTLER_BUILD_TOOL = "rattler-build"
 
+VALID_PYTHON_BUILD_BACKENDS = [
+    "setuptools",
+    "flit-core",
+    "hatchling",
+    "poetry-core",
+    "pdm-backend",
+    "pdm-pep517",
+    "meson-python",
+    "scikit-build-core",
+    "maturin",
```

Comment:
There is some code in https://venthur.de/2024-01-26-build-backends.html to query all used build backends. The only obvious one missing from their results was `jupyter`.
```suggestion
    "maturin",
    "jupyter_packaging",
```